install/installer/cluster_dev.py

```diff
@@ -283,21 +418,15 @@ def get_git_username(cli_arg, git):
     return user
 
 
-def get_git_password(cli_arg):
+@typechecked
+def get_git_password() -> {str, bool}:
```

Comment:
It's a bad idea to return `bool`. You could try returning `None` or just raising an exception (which is even better).